src/app/zap-templates/zcl/data-model/chip/test-cluster.xml

```diff
@@ -111,46 +111,84 @@ limitations under the License.
     <!-- Test Commands -->
     <command source="client" code="0x00" name="Test" optional="false">
       <description>
-          Simple command without any parameters and without a specific response
+        Simple command without any parameters and without a specific response
       </description>
     </command>
 
     <command source="client" code="0x01" name="TestNotHandled" optional="false">
       <description>
-          Simple command without any parameters and without a specific response not handled by the server
+        Simple command without any parameters and without a specific response not handled by the server
       </description>
     </command>
 
     <command source="client" code="0x02" name="TestSpecific" optional="false">
       <description>
-          Simple command without any parameters and with a specific response
+        Simple command without any parameters and with a specific response
       </description>
     </command>
 
     <command source="client" code="0x03" name="TestUnknownCommand" optional="true">
       <description>
-          Simple command that should not be added to the server.
+        Simple command that should not be added to the server.
       </description>
     </command>
 
     <command source="client" code="0x04" name="TestAddArguments" optional="false">
       <description>
-          Command that takes two arguments and returns their sum.
+        Command that takes two arguments and returns their sum.
       </description>
       <arg name="arg1" type="INT8U"/>
       <arg name="arg2" type="INT8U"/>
     </command>
 
+    <command source="client" code="0x05" name="TestSimpleArgumentRequest" optional="false">
+      <description>
+        Command that takes an argument which is bool
+      </description>
+      <arg name="arg1" type="BOOLEAN"/>
```

Comment:
How does the return value depend on this boolean?  It might make more sense to have the type in request and response be the same so we can check that the value round-trips.

Reply:
Done, thanks